python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -58,9 +58,9 @@ def getReplicas(lfns):
     output(dirac.getReplicas(lfns, active=True, preferDisk = True))
 
 
-def getAccessURL(lfn,SE):
-    ''' Return the access URL for the given LFN and storage element '''
-    output(dirac.getAccessURL(lfn,SE))
+def getAccessURL(lfn, SE, protocol):
+    ''' Return the access URL for the given LFN, storage element and protocol '''
```

Comment:
But be careful we can't use `"""` in this file due to how the file is used. It's a slightly odd limitation which is due to the way we construct Dirac scripts which would be nice to clean up at some point but is very low priority.